tests/common/db/accounts.py

```diff
@@ -30,6 +30,7 @@ class Meta:
     is_superuser = False
     is_moderator = False
     is_psf_staff = False
+    has_oidc_beta_access = True
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏Looking at this pattern, it's unclear why these are necessary, since the defaults are false. Something to look into at some future testing and Factory behaviors effort.